sigtracker/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from eveonline.models import EveCharacter
+from eveonline.models import EveCorporationInfo
+
+
+class sigtracker(models.Model):
```

Comment:
I think this model could really benefit from a "created on" `DateTimeField` with `auto_now_add=True` to track the age of entries.